tests/data/placebo/test_ecs_service_update/ecs.UpdateService_1.json

```diff
@@ -0,0 +1,329 @@
+                        "second": 9,
+                        "microsecond": 796000
+                    },
+                    "updatedAt": {
+                        "__class__": "datetime",
+                        "year": 2019,
+                        "month": 2,
+                        "day": 24,
+                        "hour": 15,
+                        "minute": 59,
+                        "second": 9,
+                        "microsecond": 796000
+                    },
+                    "launchType": "FARGATE",
+                    "platformVersion": "1.3.0",
+                    "networkConfiguration": {
+                        "awsvpcConfiguration": {
+                            "subnets": [
+                                "subnet-037318c577d65d330"
+                            ],
+                            "securityGroups": [
+                                "sg-0669346d29dd0345a"
+                            ],
+                            "assignPublicIp": "DISABLED"
+                        }
+                    }
+                },
+                {
+                    "id": "ecs-svc/9223370485813623167",
+                    "status": "ACTIVE",
+                    "taskDefinition": "arn:aws:ecs:us-east-1:644160558196:task-definition/ecs-read-only-root:1",
+                    "desiredCount": 1,
+                    "pendingCount": 0,
+                    "runningCount": 0,
+                    "createdAt": {
+                        "__class__": "datetime",
+                        "year": 2019,
+                        "month": 2,
+                        "day": 24,
+                        "hour": 15,
+                        "minute": 45,
+                        "second": 52,
+                        "microsecond": 615000
+                    },
+                    "updatedAt": {
+                        "__class__": "datetime",
+                        "year": 2019,
+                        "month": 2,
+                        "day": 24,
+                        "hour": 15,
+                        "minute": 45,
+                        "second": 52,
+                        "microsecond": 615000
+                    },
+                    "launchType": "FARGATE",
+                    "platformVersion": "1.3.0",
+                    "networkConfiguration": {
+                        "awsvpcConfiguration": {
+                            "subnets": [
+                                "subnet-037318c577d65d330"
+                            ],
+                            "securityGroups": [
+                                "sg-0fedadd839cd77f33"
+                            ],
+                            "assignPublicIp": "ENABLED"
+                        }
+                    }
+                }
+            ],
+            "roleArn": "arn:aws:iam::644160558196:role/aws-service-role/ecs.amazonaws.com/AWSServiceRoleForECS",
+            "events": [
+                {
```

Comment:
events for a service also contains alot of extraneous data that can be removed post recording.